python/docs/source/guide/quickstart.md

```diff
@@ -42,6 +42,6 @@ content = "\n".join(
         "1996-12-19T16:40:02,A,,",
     ]
 )
-source = await kd.sources.CsvString.create(content, time_column="time", key_column="key")
+source = await kd.sources.CsvString(content, time_column="time", key_column="key")
```

Comment:
I don't think this will work -- the constructor isn't async, so you can't `await` it, and the constructor doesn't accept content.